apps/www/components/Access/Campaigns/ShareChart.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { gql } from '@apollo/client'
+import compose from 'lodash/flowRight'
+import { graphql } from '@apollo/client/react/hoc'
+import { ChartTitle, ChartLead, Chart, Loader } from '@project-r/styleguide'
+import { timeDay } from 'd3-time'
+import { swissTime } from '../../../lib/utils/format'
+import withT from '../../../lib/withT'
+
+const accessGrantQuery = gql`
+  query accessGrantQuery($accessCampaignId: ID!, $min: Date!, $max: Date!) {
+    accessGrantStats {
+      evolution(accessCampaignId: $accessCampaignId, min: $min, max: $max) {
+        buckets {
+          date
+          activeUnconverted
+          converted
+        }
+        updatedAt
+      }
+      events(accessCampaignId: $accessCampaignId, min: $min, max: $max) {
+        buckets {
+          pledges
+        }
+        updatedAt
+      }
+    }
+  }
+`
+
+const formatDate = swissTime.format('%d.%m.%Y')
+
+const ShareChart = ({ data, t }) => {
+  return (
+    <div style={{ margin: '16px 0px 32px 0px' }}>
+      <Loader
+        loading={data.loading}
+        error={data.error}
+        render={() => {
+          if (!data.accessGrantStats) return null
+          const accessGrantData = [
+            { type: 'activeUnconverted', label: 'Aktiv geteilte Abos' },
+            { type: 'converted', label: 'Verkaufte Abos' },
+          ]
+            .map((key) => {
+              return data.accessGrantStats.evolution.buckets.map((bucket) => {
+                return {
+                  date: bucket.date,
+                  type: key.label,
+                  value: bucket[key.type],
+                }
```

Comment:
Move to translation for easy correction workflow and to share with color map below:
```suggestion
          const accessGrantData = [
            'activeUnconverted',
            'converted'
          ]
            .map((key) => {
              return data.accessGrantStats.evolution.buckets.map((bucket) => {
                return {
                  date: bucket.date,
                  type: t(`Share/chart/labels/${key}`) ,
                  value: bucket[key],
                }
```